tests/waku_rln_relay/test_wakunode_rln_relay.nim

```diff
@@ -11,12 +11,35 @@ import
   libp2p/protocols/pubsub/pubsub
 import
   waku/[waku_core, waku_node, waku_rln_relay],
-  ../testlib/wakucore,
-  ../testlib/wakunode,
+  ../testlib/[wakucore, futures, wakunode],
   ./rln/waku_rln_relay_utils
 
 from std/times import epochTime
 
+proc buildWakuRlnConfig_versionAware(
```

Comment:
don't think we need it to be versionAware now :)
```suggestion
proc buildWakuRlnConfig(
```

Reply:
True, thanks for the heads up.